### DISCUSSION TITLE
Welcome to USBH_MIDI Discussions!

### DISCUSSION THREAD

**Discussion by YuuichiAkagawa** [YuuichiAkagawa/USBH_MIDI#67] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomcarlotan** [comment]
Good day, I'm trying to make a controller for a guitar effect processor, specifically the Nux MG300 and MG30.
I'd like to auto detect the device but unfortunately they both have the same Vendor and Product ID. Is there a function where I can call the Product Name itself?
![NUX MG30 SS](https://user-images.githubusercontent.com/52518136/147540631-85a57bdc-74fb-4bc1-a026-badab18c148a.jpg)

**Reply by YuuichiAkagawa** [reply]
Oh, It's a badly behaved device.
This library does not provide the ability to get string descriptors, but USB_MIDI_desc.ino may be helpful.

**Reply by tomcarlotan** [reply]
Thanks for responding. Indeed, I find it weird that 2 devices have the same Product ID. Maybe they are saving on licensing fees? :) Anyway, thanks for the tip, will check the USB_MIDI_desc.ino.

**Comment by irwani123456** [comment]
ok thanks, i will try ini my Zoom G3x
